JetNews/app/src/main/java/com/example/jetnews/ui/article/PostContent.kt

```diff
@@ -125,7 +126,10 @@ private fun PostHeaderImage(post: Post) {
 private fun PostMetadata(metadata: Metadata) {
     val typography = MaterialTheme.typography()
     Row {
-        VectorImage(id = R.drawable.ic_account_circle_black)
+        VectorImage(
+            id = R.drawable.ic_account_circle_black,
+            tint = MaterialTheme.colors().onSurface
```

Comment:
Should the tint be `contentColor()` here?

Reply:
Not sure what the design intent was originally this so I went for final result. Happy to change.